cyences_app_for_splunk/default/data/ui/views/cs_data_reviewer.xml

```diff
@@ -0,0 +1,164 @@
+      <label>Product</label>
+      <search>
+        <query>| cyencesproductmanager operation="getproducts" 
+| table products</query>
+        <earliest>1</earliest>
+        <latest>now</latest>
+      </search>
+      <fieldForLabel>products</fieldForLabel>
+      <fieldForValue>products</fieldForValue>
+      <change>
+        <condition match="isnull($product_name$)">
+          <unset token="tkn_host_reviewer_search"></unset>
+          <unset token="tkn_metadata_count_search"></unset>
+          <unset token="tkn_sources_reviewer_search"></unset>
+        </condition>
+      </change>
+    </input>
+    <input type="time" token="timeRange" searchWhenChanged="true">
+      <label>Time</label>
+      <default>
+        <earliest>-24h@h</earliest>
+        <latest>now</latest>
+      </default>
+    </input>
+  </fieldset>
+  <row depends="$hidden$">
+    <panel>
+      <table>
+        <search>
+          <done>
+            <set token="tkn_host_reviewer_search">$result.host_reviewer_search$</set>
+            <set token="tkn_metadata_count_search">$result.metadata_count_search$</set>
+            <set token="tkn_sources_reviewer_search">$result.sources_reviewer_search$</set>
+          </done>
+          <query>| cyencesproductmanager operation="buildproductspecificsearches" product_name=$product_name|s$ | table host_reviewer_search, metadata_count_search, sources_reviewer_search</query>
+          <earliest>$timeRange.earliest$</earliest>
+          <latest>$timeRange.latest$</latest>
+        </search>
+        <option name="drilldown">none</option>
+        <option name="refresh.display">progressbar</option>
+      </table>
+    </panel>
+  </row>
+  <row>
+    <panel>
+      <single>
+        <title>Hosts</title>
+        <search>
+          <query>$tkn_metadata_count_search$ by host 
+| stats count</query>
+          <earliest>$timeRange.earliest$</earliest>
+          <latest>$timeRange.latest$</latest>
+        </search>
+        <option name="drilldown">none</option>
+        <option name="refresh.display">progressbar</option>
+      </single>
+    </panel>
+    <panel>
+      <single>
+        <title>Sources</title>
+        <search>
+          <query>$tkn_metadata_count_search$ by source 
+| stats count</query>
+          <earliest>$timeRange.earliest$</earliest>
+          <latest>$timeRange.latest$</latest>
+        </search>
+        <option name="drilldown">none</option>
+        <option name="refresh.display">progressbar</option>
+      </single>
+    </panel>
+    <panel>
+      <single>
+        <title>SourceTypes</title>
+        <search>
+          <query>$tkn_metadata_count_search$ by sourcetype 
+| stats count</query>
+          <earliest>$timeRange.earliest$</earliest>
+          <latest>$timeRange.latest$</latest>
+        </search>
+        <option name="drilldown">none</option>
+        <option name="refresh.display">progressbar</option>
+      </single>
+    </panel>
+  </row>
```

Comment:
Add new row with 3 tables:
* list of hosts
* list of source
* list of sourcetype

(max row count per page 5)